semantics/language/translation/init.k

```diff
@@ -260,5 +261,26 @@ module C-TRANSLATION-INIT
      syntax KItem ::= "reportSuccess"
      rule <k> reportSuccess => .K ...</k>
           <result-value> _ => 0 </result-value>
+          <generated-annotations> .List </generated-annotations>
+     rule <k> reportSuccess => writeSpecModule(#open(S +String ".spec.k")) ...</k>
```

Comment:
we could do that, but that will lead to files called foo.c_spec.k or foo.h_spec.k, which I am inclined to think looks worse than foo.c.spec.k or foo.h.spec.k.